pkg/monitoring/domainstats/prometheus/prometheus.go

```diff
@@ -67,6 +67,36 @@ func tryToPushMetric(desc *prometheus.Desc, mv prometheus.Metric, err error, ch
 	ch <- mv
 }
 
+func (metrics *vmiMetrics) updateMigrateInfo(jobInfo *stats.DomainJobInfo) {
+	if jobInfo.DataRemainingSet {
+		metrics.pushCommonMetric(
+			"kubevirt_migrate_vmi_data_remaining_bytes",
+			"The remaining VM data to be migrated.",
+			prometheus.GaugeValue,
+			float64(jobInfo.DataRemaining)*1024,
```

Comment:
Is there a reason we need this metric in bytes? Can't this multiplication be avoided?
Maybe we better use `kubevirt_migrate_vmi_data_remaining_kb`

Reply:
Yes, actually the *1024 here is an error, I removed from other places and missed this.
libvirt returns this info in bytes not kb.
And that is good because you want to have it in bytes - otherwise it gets very confusing when you look at the graph for this metrics in prometheus ui - it does not know which units you use, so you see a value like 8G and you naturally think it is 8Gb but if the metrics is in anything but bytes, you need to calculate to get the actual value. So having this in bytes is better, but no need to multiply - tnx for catching this.

Reply:
Got you. Thanks for the explanation!